x-pack/elastic-agent/spec/metricbeat.yml

```diff
@@ -1,101 +1,98 @@
 name: Metricbeat
 cmd: metricbeat
 args: [
-  "-E", "setup.ilm.enabled=false",
-  "-E", "setup.template.enabled=false",
-  "-E", "management.enabled=true",
-  "-E", "logging.level=debug",
-  "-E", "gc_percent=${METRICBEAT_GOGC:100}"
+    "-E", "setup.ilm.enabled=false",
+    "-E", "setup.template.enabled=false",
+    "-E", "management.enabled=true",
+    "-E", "logging.level=debug",
+    "-E", "gc_percent=${METRICBEAT_GOGC:100}",
+    "-E", "metricbeat.config.modules.enabled=false"
```

Comment:
@michalpristas When I initially started this change I was looking for a rule `add_config` or similar that the only thing this would od is adding this field to the config. Instead it seems all these configs we pass in as cmd line flags. What is the reason for this? 

The concerns I have with it is that when I use `inspect`, not sure if the above ones show up and at one point, we might hit some limits on the length of params.

Can you maybe share a bit of background on why we did it this way?

Reply:
we did this initially to enable management and later on to set debug level nothing else. all the other options here are just using the stuff we introduced initially to have management enabled. 
add_config_option sound good but we need to make it clear what to do in case of naming conflict. something like we have with copy rule where we specify overwrite/merge/noop

Reply:
Managemement and logging can also be set as config options, or does management have to be passed in as environment variable?